README.md

```diff
@@ -1,214 +1,282 @@
 
-CRDTs, on the other hand, appear more user-friendly. They offer built-in support for fundamental data structures (e.g., texts, lists, maps, counters), reducing the workload for developers. However, CRDTs often introduce metadata overhead and become challenging to manage in scenarios involving partial syncing or permission controls—such as when users can only access a subset of files. These issues arise because CRDTs are primarily designed for decentralized systems, while most modern applications still depend on centralized servers. Although I am personally an advocate of CRDTs, they often fall short for some practical use cases.
+CRDTs, on the other hand, appear more user-friendly. They offer built-in support for fundamental data structures (e.g., texts, lists, maps, counters), reducing the workload for developers. However, CRDTs often introduce metadata overhead and become challenging to manage in scenarios involving partial syncing or permission controls—such as when users can only access a subset of files. These issues arise because CRDTs are primarily designed for decentralized systems, while most modern applications still depend on centralized servers. Although I am personally an advocate of CRDTs, they often fall short for some use cases.
 
 What I look for in a sync engine includes:
 
-1. **Support for arbitrary data models**: Not limited to rich text or spreadsheets, making it adaptable to a wide range of scenarios.
+1. **Support for arbitrary data models**: Making it adaptable to a wide range of scenarios.
 2. **Rich features**: It should support partial syncing, enforce permission control, and include features like undo/redo, offline availability, and edit history.
-3. **Great developer experience**: Ideally, it should allow model definitions in an ORM-like manner. Developers should not need to be experts in the engine to build collaborative applications.
+3. **Great developer experience**: Ideally, it should allow model definitions in an ORM-like manner. Developers should not need to be experts in sync engines to build collaborative applications.
 
-Then, I discovered [Linear](https://linear.app). The [Linear Sync Engine (LSE)](https://linear.app/docs/offline-mode) offers an elegant solution to all the requirements mentioned above. Additionally, it provides a streamlined API that simplifies feature development. For example, updating the title of an issue is as simple as:
+[Linear](https://linear.app)'s [Linear Sync Engine (LSE)](https://linear.app/docs/offline-mode) provides an elegant solution to all the aforementioned requirements. Moreover, it offers an intuitive API that abstracts away the underlying complexity, making feature development significantly simpler. For instance, updating the title of an issue can be as straightforward as:
```

Comment:
This link `https://linear.app/docs/offline-mode` appears to not work btw.

Great work here, thanks for making it!